app/models/movie.rb

```diff
@@ -10,22 +11,16 @@ class Movie < ActiveRecord::Base
   has_many :listings
   has_many :lists, through: :listings
   has_many :users, through: :listings
-
-  def self.by_tag_and_user(tag, user)
-    joins(:taggings).where(taggings: { user_id: user.id }).where(taggings: { tag_id: tag.id })
-  end
-
   has_many :taggings
   has_many :tags, through: :taggings
-
   has_many :reviews
-
   has_many :ratings
-
   has_many :screenings
   has_many :viewers, :through => :screenings,
   :source => :user
 
+  attr_accessor :production_companies
```

Comment:
Except for this. 😆 This bridges the gap between the `Movie` class and the `MovieMore` class. The `production_companies` previously lives as an instance variable in the `tmdb_handler_movie_more` method and I tucked it into the `MovieMore` object. This attr_accessor implementation doesn't render data for a `Movie` object. I made issue #278 to address it.